HW1/task4.py

```diff
@@ -0,0 +1,14 @@
+#Задача 8: Требуется определить, 
+# можно ли от шоколадки размером n × m долек отломить k долек, 
+# если разрешается сделать один разлом по прямой между дольками (то есть разломить шоколадку на два прямоугольника).
+
+print('Введите размер одной стороны шоколадки: ')
+n = int(input())
+print('Введите размер второй стороны шоколадки: ')
+m = int(input())
+print('Введите сколько долек необходимо: ')
+k = int(input())
+if k < m * n and k % m == 0 or k % n == 0:
```

Comment:
Условия верные, но здесь нужны скобки:
```suggestion
if k < m * n and (k % m == 0 or k % n == 0):
```
К тому же первое условие можно расширить относительно нижней границы допустимых значений переменной `k`.